scripts/install_ostis.sh

```diff
@@ -26,6 +26,7 @@ include_kpm()
   echo 'add_subdirectory(${SC_MACHINE_ROOT}/../../problem-solver/cxx ${SC_MACHINE_ROOT}/bin)' >> ./CMakeLists.txt
 	cd ./scripts
 	./make_all.sh
+        ./install_interface_comic.sh
```

Comment:
it's better to move this call in separate function:
- Create function 'install_interface_comic' and call this script inside.
- Add function call after include_kpm (line 48)